### PR TITLE
Widen zipcode input: #290

### PR DESCRIPTION
**Pull request by cecilia-donnelly**
Show that the zipcode accepts 10 digits (zip plus 4, with the hyphen).
Narrow the county and state dropdowns to keep all these on one line.

I checked these on the individual and group provider applications, and from the perspective of an admin.

Issue #290 Add 4 extra digits to zipcode field(s)

### CODE REVIEW

File: psm-app/cms-web/WebContent/css/style.css [hunk 1]
```diff
@@ -4932,7 +4932,15 @@ td.nopad {
 }
 
 input.zipInput, input.zipInputFor {
```
Comment (jasonaowen) [inline psm-app/cms-web/WebContent/css/style.css]: After talking on IRC, it seems to @cecilia-donnelly and me that having both `zipInput` and `zipInputFor` classes doesn't make a lot of sense. They seem to be used to distinguish between practice addresses and billing/reimbursement addresses, and IDs or additional, more specific classes would better solve that problem.

**Comment by cecilia-donnelly** [comment]
@jasonaowen, take another look at this and let me know what you think.

**Comment by cecilia-donnelly** [comment]
I removed the final references to `zipInput`, including deleting the two functions that were meant to update the values in the `zipInput` fields.  As @jasonaowen writes, those don't work right now.  We can replace/fix them later if we need to.

File: psm-app/cms-web/WebContent/css/style.css [hunk 2]
```diff
@@ -1767,7 +1767,7 @@ a.purpleSmallBtn:hover{
 				width:350px;
 				}
 				
-			.affiliationPanel table tr td .zipInput{
+			.affiliationPanel table tr td {
```
Comment (jasonaowen) [inline psm-app/cms-web/WebContent/css/style.css]: I think we need to delete this block - removing the `.zipInput` widens its scope by changing the selector from 'an element with the `zipInput` class that is a child of a `td` element that is the child of a `tr` element...' to 'a `td` element that is the child of a `tr` element...'.

Comment (cecilia-donnelly) [reply psm-app/cms-web/WebContent/css/style.css]: Wow, right, of course.  Will do!

**Pull request merged**